### PR TITLE
LPS-169795 Create TagsInfoFilter

### PR DESCRIPTION
**Pull request by p2kmgcl**
This is an initial development for the story, adding a new "Tags" filter fragment. The component is still a dummy fragment that does nothing. In order to test this:

1. Enable FragmentCollectionFilters component.
2. Deploy all modules.
3. Create a manual collection of Web Content Articles (it can be empty).
4. Add a content page with:
   - A collection display fragment targeting the created collection.
   - A collection filter fragment targeting the created collection display.
5. Select "tags" in filter type and publish.

Things to check:
- The page can be published.
- If something is typed into the fragment and enter is pressed, the page reloads and a new URL parameter appears.
- The fragment preserves typed tags into the URL.

In future PRs I will add:
- Autocomplete with existing tags
- Actual filtering of elements

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-collection-filter-tags/src/main/java/com/liferay/fragment/collection/filter/tags/FragmentCollectionFilterTags.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	}
+
+	@Override
+	public String getFilterKey() {
+		return "tags";
+	}
+
+	@Override
+	public String getFilterValueLabel(String filterValue, Locale locale) {
+		AssetCategory assetCategory =
```
Comment (ruben-pulido) [inline modules/apps/fragment/fragment-collection-filter-tags/src/main/java/com/liferay/fragment/collection/filter/tags/FragmentCollectionFilterTags.java:83]: The implementation of this method is using AssetCategories, probably a copy-paste error

File: modules/apps/fragment/fragment-collection-filter-tags/src/main/java/com/liferay/fragment/collection/filter/tags/FragmentCollectionFilterTags.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+			_log.error(
+				"Unable to render collection filter fragment", exception);
+		}
+	}
+
+	private static final Log _log = LogFactoryUtil.getLog(
+		FragmentCollectionFilterTags.class);
+
+	@Reference
+	private AssetCategoryLocalService _assetCategoryLocalService;
```
Comment (ruben-pulido) [inline modules/apps/fragment/fragment-collection-filter-tags/src/main/java/com/liferay/fragment/collection/filter/tags/FragmentCollectionFilterTags.java:127]: We'll need AssetTagLocalService instead

File: modules/apps/fragment/fragment-collection-filter-tags/src/main/java/com/liferay/fragment/collection/filter/tags/FragmentCollectionFilterTags.java (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/fragment/fragment-collection-filter-tags/src/main/java/com/liferay/fragment/collection/filter/tags/FragmentCollectionFilterTags.java]: Done!

**Comment by p2kmgcl** [comment]
ci:close

**Pull request closed without merging**